signer-service/src/api/services/sep10.service.js

```diff
@@ -88,8 +123,15 @@ exports.signSep10Challenge = async (challengeXDR, outToken, clientPublicKey) =>
     clientDomainSignature = transactionSigned.getKeypairSignature(clientDomainStellarKeypair);
   }
 
+  let masterClientSignature;
+  if (memo !== null && memoEnabled) {
```

Comment:
We've verified all operations so reaching here should be safe to append the signature.